test/tests.transform.js

```diff
@@ -47,3 +47,102 @@ describe("_blockHoist nodes", function() {
     assert.deepEqual(names, ["hoistMe", "doNotHoistMe", "oyez"]);
   });
 });
+
+context("functions", function() {
```

Comment:
> in our case we're generating api sagas automatically from swagger docs and grouping them by model.

metaprogramming level hard :D would love to see this in action

> although i can imagine situations with code-spitting where you end up bundling an entire module because you want access to a single saga inside of it

yeah, thats true, however i see the issue only for 'util' saga files, others are rather grouped by some kind of functionality and they rather work only together